master/internal/job/jobservice/jobservice.go

```diff
@@ -187,6 +187,11 @@ func (s *Service) applyUpdate(update *jobv1.QueueControl) error {
 	if j == nil {
 		return sproto.ErrJobNotFound(jobID)
 	}
+	job := s.jobByID[jobID]
+	v1Job, err := job.ToV1Job()
```

Comment:
can we just add `ResourcePool() string` to this interface and then call `j.ResourcePool()` where we need it?

if not that, we don't need to do jobByID again, we can use j from 3 lines above us

Reply:
Gahh, totally spaced on the previous `j` assignment. 
Will add `GetResourcePool()` to the `Job` interface!

Reply:
https://go.dev/doc/effective_go#Getters